src/pages/bioclimatic/component.jsx

```diff
@@ -58,6 +59,7 @@ function BioClimaticPage(props) {
               </LayerManager>
             )}
           </Map>
+          <Timeline className="timeline" activeTab={scenario} data={timelineData} hideHeader />
```

Comment:
Yes, I described it in `known issues` section above, I was going to leave it on another PR, but then I discovered that I can use `formatValue` function to change this value in the tooltip from index to year. The best part is that by extending this function I can fix another problem with the timeline animation (after the animation stops, the current year is the last one, instead of the first). Maybe it's a bit hacky way of doing things, but it is the quickest solution that we can do right now.
![image](https://user-images.githubusercontent.com/15097138/68233400-0789ff00-fff7-11e9-85f4-696baa4b466a.png)

Reply:
Sorry I missed the `known issues` part. I think it looks good now!